TypewiseAlert.Test/TypewiseAlertTest.cs

```diff
@@ -1,15 +1,91 @@
+using Moq;
 using System;
 using Xunit;
 
 namespace TypewiseAlert.Test
 {
-  public class TypewiseAlertTest
-  {
-    [Fact]
-    public void InfersBreachAsPerLimits()
+    public class TypewiseAlertTest
     {
-      Assert.True(TypewiseAlert.inferBreach(12, 20, 30) ==
-        TypewiseAlert.BreachType.TOO_LOW);
+        [Fact]
```

Comment:
Is it ok to have multiple assertion in single unit test case ?

Reply:
Made single assertion in the new commit sir